lib/process-functional.js

```diff
@@ -1,11 +1,16 @@
 module.exports = function extractProps (content) {
```

Comment:
Can you rename this file to extract-props, so that it matches the named exported function

Reply:
Done!